src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/BackpackCommand.java

```diff
@@ -0,0 +1,72 @@
+package io.github.thebusybiscuit.slimefun4.core.commands.subcommands;
+
+import io.github.thebusybiscuit.slimefun4.api.player.PlayerProfile;
+import io.github.thebusybiscuit.slimefun4.core.commands.SlimefunCommand;
+import io.github.thebusybiscuit.slimefun4.core.commands.SubCommand;
+import io.github.thebusybiscuit.slimefun4.utils.PatternUtils;
+import me.mrCookieSlime.Slimefun.Lists.SlimefunItems;
+import me.mrCookieSlime.Slimefun.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.api.Slimefun;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.List;
+
+class BackpackCommand extends SubCommand {
+
+    BackpackCommand(SlimefunPlugin plugin, SlimefunCommand cmd) {
+        super(plugin, cmd);
+    }
+
+    @Override
+    public String getName() {
+        return "backpack";
+    }
+
+    @Override
+    public boolean isHidden() {
+        return false;
+    }
+
+    @Override
+    public void onExecute(CommandSender sender, String[] args) {
+        if (!(sender instanceof Player) || !sender.hasPermission("slimefun.command.backpack")) {
+            SlimefunPlugin.getLocal().sendMessage(sender, "messages.no-permission", true);
+            return;
+        }
+        if (args.length != 3) {
+            SlimefunPlugin.getLocal().sendMessage(sender, "messages.usage", true,
+                    msg -> msg.replace("%usage%", "/sf backpack <Player> <ID>"));
+            return;
+        }
+
+        final Player p = (Player) sender;
+        if (!PatternUtils.NUMERIC.matcher(args[2]).matches()) {
+            SlimefunPlugin.getLocal().sendMessage(sender, "commands.backpack.invalid-id");
+            return;
+        }
+        final int id = Integer.parseInt(args[2]);
+        final OfflinePlayer owner = Bukkit.getOfflinePlayer(args[1]);
+        if (!owner.hasPlayedBefore()) {
+            SlimefunPlugin.getLocal().sendMessage(sender, "commands.backpack.player-never-joined");
+            return;
+        }
+
+        PlayerProfile.get(owner, profile -> {
+            if (!profile.getBackpack(id).isPresent()) {
+                SlimefunPlugin.getLocal().sendMessage(sender, "commands.backpack.backpack-does-not-exist");
+                return;
+            }
+            Slimefun.runSync(() -> {
+                ItemStack item = SlimefunItems.RESTORED_BACKPACK;
```

Comment:
Hold up...
Very dangerous here, you need to add a .clone() otherwise you are modifying the original and not a copy.